templates/zerver/typing_notifications.html

```diff
@@ -0,0 +1,2 @@
+<div id="typing_notifications">
+</div>
```

Comment:
I think you don't need this template; just put the div directly in the HTML (just how `bottom_whitespace` works).